apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.html

```diff
@@ -0,0 +1,16 @@
+<div class="ba-uxd-startnode-grid">
+  <div
+    class="ba-uxd-startnode-wrapper"
+    *ngFor="let startnode of decisionGraphStartnodes, let i = index"
+  >
+    <div
+      class="ba-uxd-startnode"
+      [ngClass]="{
+        'dt-uxd-startnode-visited': isSelectedStartnode(i)
+      }"
```

Comment:
if you bind only one property please use normal class binding
```suggestion
      [class.dt-uxd-startnode-visited]="isSelectedStartnode(i)"
```